src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/ExportTiles/ExportTiles.xaml.cs

```diff
@@ -0,0 +1,264 @@
+            GraphicsOverlay extentOverlay = MyMapView.GraphicsOverlays.FirstOrDefault();
+
+            // Get the area selection graphic's extent
+            Graphic extentGraphic = extentOverlay.Graphics.FirstOrDefault();
+
+            // Set the area for the export
+            parameters.AreaOfInterest = extentGraphic.Geometry;
+
+            // Get the highest possible export quality
+            parameters.CompressionQuality = 100;
+
+            // Add level IDs 1-9
+            //     Note: Failing to add at least one Level ID will result in job failure
+            for (int x = 1; x < 10; x++)
+            {
+                parameters.LevelIds.Add(x);
+            }
+
+            // Return the parameters
+            return parameters;
+        }
+
+        /// <summary>
+        /// Called by the ExportTileCacheJob on any status changes
+        /// </summary>
+        private void Job_JobChanged(object sender, EventArgs e)
+        {
+            // Get reference to the job
+            ExportTileCacheJob job = sender as ExportTileCacheJob;
+
+            // Update the view if the job is complete
+            if (job.Status == Esri.ArcGISRuntime.Tasks.JobStatus.Succeeded)
+            {
+                // Dispatcher is necessary due to the threading implementation;
+                //     this method is called from a thread other than the UI thread
+                this.Dispatcher.Invoke(() =>
+                {
+                    // Show the exported tiles on the preview map
+                    UpdatePreviewMap();
+
+                    // Show the preview window
+                    MyPreviewMapView.Visibility = Visibility.Visible;
+
+                    // Hide the progress bar
+                    MyProgressBar.Visibility = Visibility.Collapsed;
+                });
+            }
+            else if (job.Status == Esri.ArcGISRuntime.Tasks.JobStatus.Failed)
+            {
+                // Notify the user
+                ShowStatusMessage("Job failed");
+
+                // Dispatcher is necessary due to the threading implementation;
+                //     this method is called from a thread other than the UI thread
+                this.Dispatcher.Invoke(() =>
+                {
+                    // Hide the progress bar
+                    MyProgressBar.Visibility = Visibility.Collapsed;
+                });
+            }
+        }
+
+        /// <summary>
+        /// Loads the tile cache from disk and displays it in the preview map
+        /// </summary>
+        private async void UpdatePreviewMap()
+        {
+            // Get the path to the cache
+            String filePath = GetTilePath();
+
+            // Load the saved tile cache
+            TileCache cache = new TileCache(filePath);
+
+            // Load the cache
+            await cache.LoadAsync();
+
+            // Create a tile layer with the cache
+            ArcGISTiledLayer layer = new ArcGISTiledLayer(cache);
```

Comment:
Don't name variables the same as ArcGIS Runtime .NET classes. It should be something like 'myLayer' not 'layer'.